web/src/views/org/settings/OrgSettingsWrapper.vue

```diff
@@ -11,17 +11,11 @@
       </span>
     </template>
 
-    <Tab id="secrets" :title="$t('secrets.secrets')">
-      <OrgSecretsTab />
-    </Tab>
+    <Tab to="org-settings-secrets" :title="$t('secrets.secrets')" />
```

Comment:
Would suggest to use Vue's `:to="{ name: 'org-settings-secrets'}` syntax instead.

Reply:
That has to be possible somehow. When using `<router-link :to="...">` it also detects if the route is [currently active](https://router.vuejs.org/guide/essentials/active-links#Active-links) and sets `router-link-active` and `router-link-exact-active` respectively. That would make the syntax a bit cleaner.